py/private/py_wheel.bzl

```diff
@@ -24,7 +24,11 @@ def _make_py_wheel_info(ctx, wheel_filegroups):
         if PyWheelInfo in filegroup:
             files_depsets.append(filegroup[PyWheelInfo].files)
             runfiles.append(filegroup[PyWheelInfo].default_runfiles)
-        elif DefaultInfo in filegroup:
+        elif DefaultInfo in filegroup and not PyInfo in filegroup:
+            # This is slightly incorrect, but we don't yet have a better way of knowing if the dependency is a filegroup
+            # that we should consume a wheel from.
```

Comment:
file an issue on rules_python, if we think the right fix is to put more Info on these targets?

Reply:
Yeah ideally the `PyWheelInfo` is removed from rules_py and added to rules_python, and the filegroup in the generated BUILD files for PyPi dependencies become the `py_wheel` rule.